crates/astria-composer/src/executor/mod.rs

```diff
@@ -89,7 +90,7 @@ pub(super) struct Executor {
     // The status of this executor
     status: watch::Sender<Status>,
     // Channel for receiving `SequenceAction`s to be bundled.
-    serialized_rollup_transactions_rx: mpsc::Receiver<SequenceAction>,
+    sequence_action_rx: mpsc::Receiver<SequenceAction>,
```

Comment:
Personally, I'd just call this `sequence_actions` or new_sequence_actions`. The `_rx` doesn't add anything of substance.